src/Fable.React.Helpers.fs

```diff
@@ -173,18 +173,56 @@ module Helpers =
 
     [<Emit("typeof $0 === 'object' && !$0[Symbol.iterator]")>]
     let private isNonEnumerableObject (x: obj): bool = jsNative
+
+    [<Emit("$0.length")>]
+    let private jsArrayLength(arr: ResizeArray<_>): int = jsNative
```

Comment:
Hmm, `.Count` has some overhead (it has to checker whether the object is actually an array, although maybe we could optimize that), but if we're using so many JS helpers I wonder if it wouldn't be better to write the function directly in JS.

Reply:
Yeah I removed Count when I saw the codegen... as for writting it in JS directly it make the calls to equal a little more complex but yeah it might be easier.

I just wasn't sure how to add pure-JS code to a pure-F# library. (Especially handling server-side rendering where there should still be an empty version for the .NET F# compiler to find